### PR TITLE
WIP: [Docs] add PPO_GAE_CHUNK document

### PR DESCRIPTION
**Pull request by m1nttee**
Since slime has not yet published a Docker image, we do not provide specific installation instructions at this time.

### CODE REVIEW

File: rlhf/slime/batch-GAE/ppo-gae-chunk.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    delta = full_rewards[:, t] + gamma * next_value - full_values[:, t]
+    lastgaelam = delta + gamma * lambd * lastgaelam
+    adv_rev.append(lastgaelam)
+
+full_advantages = torch.stack(adv_rev[::-1], dim=1)  # [B, max_len]
+```
+
+而这段代码的主要问题是：它是串行地由后向前推的，无法在时间维度上并行，随着序列逐渐变长，这一步花费的时间会变得非常可观。
+
+slime 在一开始的实现里，追求的是“支持变长序列”。我们可以看到，slime 在设计之初就已经在性能方面做出了一些考虑，它改成常见的 “padding 到 max_len，再按 batch 计算 GAE” 的写法。但是可惜的是，这并不足以达到可能的最佳性能，它在时间维度仍是串行，而这导致了在长序列场景下依然很吃力。
```
Comment (lilei199908) [inline rlhf/slime/batch-GAE/ppo-gae-chunk.md]: 这句改成下面会通顺点吗？：slime 在一开始的实现里，追求的是“支持变长序列”，优点是在模型计算时，不需要padding到所有序列的max_len， 避免浪费无效的计算，所以在计算 GAE 时，是一个序列一个序列计算而不是拼成 batch 计算，造成了性能瓶颈，我们很快改成常见的 “padding 到 max_len，再按 batch 计算 GAE” 的写法。但是可惜的是，这并不足以达到可能的最佳性能，它在时间维度仍是串行，而这导致了在长序列场景下依然很吃力。

Comment (m1nttee) [reply rlhf/slime/batch-GAE/ppo-gae-chunk.md]: > 这句改成下面会通顺点吗？：slime 在一开始的实现里，追求的是“支持变长序列”，优点是在模型计算时，不需要padding到所有序列的max_len， 避免浪费无效的计算，所以在计算 GAE 时，是一个序列一个序列计算而不是拼成 batch 计算，造成了性能瓶颈，我们很快改成常见的 “padding 到 max_len，再按 batch 计算 GAE” 的写法。但是可惜的是，这并不足以达到可能的最佳性能，它在时间维度仍是串行，而这导致了在长序列场景下依然很吃力。

你这个版本确实通顺点，已修改
